instagram/src/components/CommentSection/comment-section.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+import Timestamp from './Timestamp'
+import styled from 'styled-components'
+
+const Thecontainer = styled.div`
+  width: 100%;
+  margin-bottom: 10px;
+`
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      comment: '',
+      timestamp : props.timestamp
+    };
+  }
+
+  componentDidMount() {
+    const id = this.props.postId;
+    if (localStorage.getItem(id)) {
+      this.setState({
+        comments: JSON.parse(localStorage.getItem(this.props.postId))
+      });
+    } else {
+      this.setComments();
+    }
+  }
+
+  componenetWillUnmount() {
```

Comment:
It's rare to see this being used, it's impressive to see you found a spot for it.